packages/loot-core/src/server/budget/template-notes.ts

```
@@ -114,4 +121,4 @@
   });
 
   return templatesForCategory;
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)